_posts/2024-11-07-next-lts-3-15.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: 'Our next LTS will be Quarkus 3.15'
+date: 2024-07-11
+tags: release
+synopsis: 'Our next LTS (Long Term Support) Quarkus release will be Quarkus 3.15 released at the end of September.'
+author: gsmet
+---
+
+We started releasing LTS releases with Quarkus 3.2.
+LTS releases are supported significantly longer than our regular Quarkus releases.
+
+If you don't want to follow closely the development of Quarkus,
+and if you don't want to update regularly your applications to the latest Quarkus,
+LTS releases are the best choice for you.
+
+Our second LTS was Quarkus 3.8, released in February.
+
+It is our pleasure to announce that the next LTS will be Quarkus 3.15,
+that we plan to release at the end of September.
+
+For 3.15, we will follow the same strategy we inaugurated for 3.8,
+strategy that makes sure the LTS release is rock solid from the start.
+
+[IMPORTANT]
+====
+**tl;dr**
+
+Quarkus 3.15 will be our next LTS release.
+It will be the direct continuation of the `3.14` branch.
+
+If you contribute to Quarkus or the Quarkus Platform and need a feature in the next Quarkus LTS,
+make sure it has been merged in the https://github.com/quarkusio/quarkus[Quarkus repository] before August 13th included
+(the day before the `3.14.0.CR1` release).
```

Comment:
should we put something like "After August 13th main will be what becomes Quarkus 3.16" or similar so its explicit main is for new stuff.

Reply:
It's explained in the details for each version.